docs/release-promotion/desktop/howto-rc.md

```diff
@@ -18,6 +18,7 @@ What's New Page setup should be done shortly after updates are available on the
   * Search for the release blob you need
   * Click `Download`
   * Copy it to a `-No-WNP` version (you'll need this later). Eg: `Firefox-63.0-build1-No-WNP.json`.
+  * Modify this file to have a `-No-WNP` in the name.
```

Comment:
I'd stick to 

> * Append `-No-WNP` to the `name` field.

This sounds less ambiguous to me. The current suggested wording makes me think of changing the filename (which was already done in the step before), so I would be confused if this was my first time doing it.